CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to Skunk
+Have a fix for a problem you've been running into or an idea for a new feature you think would be useful? Bug reports and pull requests are welcome on GitHub at [https://github.com/fastruby/skunk](https://github.com/fastruby/skunk).
+
+Here's what you need to do:
+
+- Read and understand the [Code of Conduct](https://github.com/fastruby/skunk/blob/main/CODE_OF_CONDUCT.md).
+- Fork this repo and clone your fork to somewhere on your machine.
+- [Ensure that you have a working environment](#setting-up-your-environment)
+- Read up on the [architecture of the gem](#architecture), and [run the tests](#running-all-tests).
+- Cut a new branch and write a failing test for the feature or bug fix you plan on implementing.
+- [Update the changelog when applicable](#a-word-on-the-changelog).
+- Read the guidelines when [Releasing a new version](#releasing-a-new-version)
+- Push to your fork and submit a pull request.
+- [Make sure the test suite passes on GitHub Actions and make any necessary changes to your branch to bring it to green.](#continuous-integration).
+- Check out the [Help Wanted](#help-wanted) section for suggestions on how to support Skunk.
+
+## Setting up your environment
+To install the dependencies, run:
+
+```bash
+bin/setup
+```
+
+You can also run `bin/console` for an interactive prompt that will allow you to experiment with the gem.
+
+To install this gem onto your local machine, run:
+
+`bundle exec rake install`.
+
+## Architecture
+
+This project follows the typical structure for a gem: code is located in `lib` and tests are in `test`.
+
+Under `/lib/skunk`, you find the CLI commands. Under `lib/skunk/rubycritic` you find skunk's extension from `rubycritic` gem. These are likely the files you'll be most interested in.
```

Comment:
This looks good, I just think this: `These are likely the files you'll be most interested in.` is not necesary

Reply:
thanks @bronzdoc I agree.